@anisheremariam looks like the linter is still reporting one trailing space issue
> E       AssertionError: Code quality issues:
E         File contains trailing whitespace: examples/basic/scatters/image_url.py, line 7.
E       assert 1 == 0
E        +  where 1 = len(['File contains trailing whitespace: examples/basic/scatters/image_url.py, line 7.'])

FYI you can run all the codebase tests to check these things locally: 

https://docs.bokeh.org/en/latest/docs/dev_guide/testing.html#run-codebase-tests

--- a/docs/bokeh/source/docs/user_guide/basic/scatters.rst
+++ b/docs/bokeh/source/docs/user_guide/basic/scatters.rst
@@ -68,6 +68,8 @@ All the markers have the same set of properties: ``x``, ``y``, ``size`` (in
 an exception: this method accepts an additional ``radius`` property that you can
 use with |data units|.
 
+.. _ug_basic_scatters_urls:
+
 Image URLs
 ----------
 

--- a/examples/basic/scatters/image_url.py
+++ b/examples/basic/scatters/image_url.py
@@ -4,7 +4,7 @@
     :apis: bokeh.plotting.figure.image_url
     :refs: :ref:`ug_basic_scatters_urls`
     :keywords: image, scatter
- 
+
 .. _Bokeh image logo: https://static.bokeh.org/logos/logo.png
 '''
 import numpy as np

--- a/examples/plotting/customjs_expr.py
+++ b/examples/plotting/customjs_expr.py
@@ -1,4 +1,4 @@
-''' This example demonstrates using CustomJS Expression to create a sinusoidal line chart with Bokeh.
+''' This example demonstrates using CustomJS Expression to create a sinusoidal line chart.
 
 .. bokeh-example-metadata::
     :apis: bokeh.models.CustomJSExpr, bokeh.model.DataModel

